options.py

```diff
@@ -0,0 +1,72 @@
+
+def options():
+    print()
+    print('1. Show all employees')
+    print('2. Create a new employees')
+    print('3. Delete an employees')
+    print('4. Country statistics')
+    print('5. Exit')
+
+
+def show_all_employees(connection_obj, width_name, width_country, width_salary, full_width):
+
+    cursor = connection_obj.cursor()
+    cursor.execute("""select e.name, 
+            case 
+                when c.country IS NULL then 'Unknown'
+                else c.country
+            end country, 
+            e.salary 'salary ($)' 
+            from employees_info e left join countries c
+            on e.country_code = c.code""")
+
+    col_names = [cn[0] for cn in cursor.description]
+
+    print(f'\n{col_names[0].center(width_name)}|{col_names[1].center(width_country)}|'
+          f'{col_names[2].center(width_salary)}|')
+    print('-' * full_width)
+
+    for name, country, salary in cursor.fetchall():
+        print(f'{name:{width_name}}|{country.center(width_country)}|{salary:{width_salary},.2f}|')
+
+
+def create_new_employees(connection_obj, name, country_code, salary):
+    cursor = connection_obj.cursor()
+    cursor.execute("""insert into employees_info values (?, ?, ?)""", [name, country_code, salary])
+    connection_obj.commit()
+
+
+def delete_employee(connection_obj, name_of_employee):
+    cursor = connection_obj.cursor()
+    cursor.execute('delete from employees_info where name=?', [name_of_employee])
+    if cursor.rowcount:
+        connection_obj.commit()
+        return
+    print('There is not such name in the table')
+
+
+def country_statistics(connection_obj, width_country):
+    query_group_by_country = """select 
+            case 
+                when c.country IS NULL then 'Unknown'
+                else c.country
+                end country, 
+            count(*) num_of_employees, 
+            sum(salary) tot_salary, max(salary) max_salary 
+            from employees_info e left join countries c 
+            on e.country_code = c.code
+            group by country;"""
+
+    cursor = connection_obj.cursor()
+    cursor.execute(query_group_by_country)
+
+    col_names = [cn[0] for cn in cursor.description]
```

Comment:
Maybe you can write some universal code that gets a cursor as a parameter and outputs a table?